### PR TITLE
Features/profile

### PR DESCRIPTION
**Pull request by Alzubair98**
the requirements for profile table is done , now the reversed rockets will be lists in the profile

### CODE REVIEW

File: src/components/Rockets.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { useSelector } from 'react-redux';
+import RocketCard from './RocketCard';
+
+const Rockets = () => {
+  const state = useSelector((state) => state.storeSlice);
+  // const dispatch = useDispatch();
+
+  // useEffect(() => {
+  //   dispatch(loadRockets());
+  // }, []);
```
Comment (karlavdelgadof) [inline src/components/Rockets.js]: - [x] Please remove commented code as it will make the code cleaner :sparkles:

File: src/components/navbar.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      id: 2,
+      path: '/Missions',
+      text: 'Missions',
+    },
+    {
+      id: 3,
+      path: 'MyProfile',
+      text: 'MyProfile',
+    },
+  ];
```
Comment (karlavdelgadof) [inline src/components/navbar.js]: - [x] There is no need to implement an array to hold the links, you could accomplish this by just adding the `path /` in the `to=/myprofile` attribute/props of the `<NavLink >` tag :heavy_check_mark:

**Review by karlavdelgadof — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES :recycle: 

Hi @Alzubair98 :smiley:  

Your profile/rockets :rocket: feature is looking awesome so far :100:, however, there are some pieces that can be improved to accomplish a more readable/clean code for collaboration :busts_in_silhouette: 



### Highlights :sparkles:

- Profile rocket display functionality is working as expected :rocket: :+1: 
- Great work using the right GitHub Flow :100: 
- Awesome file organization :card_file_box: 
- Commits are descriptive, good work. 
- Linters are working properly, great job!

Good job so far! :clap:  

### Required Changes ♻️

- [ ] Please ensure that the PRs are descriptive enough for the collaborative environment :100: 
- [ ] Please link the issue (task) related to the PR of any feature you work on :eyes: 

For more information, check the comments under the review :arrow_down: 

There are some issues that you still need to work on to go to the next project but you are almost there! 

### Optional suggestions :ballot_box_with_check: 

*Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.*

Congrats and keep it up! :raised_hands: 

*Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can do this by tagging me @karlavdelgadof in the conversation!*

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

<sub>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)</sub>

File: src/components/navbar.js (hunk 2 above)
Comment (Alzubair98) [reply src/components/navbar.js]: if we do so ! how can we show the names of the pages in the nav bar? @karlavdelgadof

File: src/components/Rockets.js (hunk 1 above)
Comment (Alzubair98) [reply src/components/Rockets.js]: @karlavdelgadof  yes you are right i will change it

File: src/components/navbar.js (hunk 2 above)
Comment (Alzubair98) [reply src/components/navbar.js]: what i made is 
im making a object inside that object there is the id and path and name of the nav link 
and them i loop on the object and create the links dynamically using map()

Comment (karlavdelgadof) [reply src/components/navbar.js]: > if we do so ! how can we show the names of the pages in the nav bar? @karlavdelgadof

You could just add it like this `<NavLink some attributes=...>My Profile</NavLink>`

Comment (karlavdelgadof) [reply src/components/navbar.js]: > what i made is im making a object inside that object there is the id and path and name of the nav link and them i loop on the object and create the links dynamically using map()

Alright @Alzubair98 we can try this way too, no problem :+1:

**Review by karlavdelgadof — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @Alzubair98 :star_struck: 

Your project is complete! There is nothing else to say other than... it's time to merge it! 🚀
Great job, congratulations! 🎉



### Highlights :sparkles:

- Profile/rockets functionality is working as expected :rocket: :+1: 
- Great work using the right GitHub Flow :100: 
- Awesome file organization :card_file_box: 
- Commits are descriptive, good work. 
- Linters are working properly, great job!

APPROVED! 💯 💯 💯


Congrats and Happy coding mate! :man_technologist: 🤓

### Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

________________________________________________________________________

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, remember to tag me @karlavdelgadof  in your question so I can receive the notification._**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
____________

**Pull request merged**